.rubocop.yml

```diff
@@ -4,6 +4,24 @@ AllCops:
   Exclude:
     - '**/vendor/**/*'
 
+Style/FrozenStringLiteralComment:
+  Enabled: true
+  EnforcedStyle: always
+  Exclude:
+    - 'example/**/*'
+    - 'test/builder/**/*'
```

Comment:
Why is this rule disabled on tests?

Reply:
@rafaelfranca Thanks for the comment. You're right. It doesn't need to set `Exclude`. So I've removed it and added `frozen_string_literal` comments to all files.